admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -765,7 +771,7 @@ def _makeIndex(self, database, table):
         self._log.info('Generating index %r.%r', self.indexDb, metaTable)
 
         # try to delete existing table first
-        self.czarWmgr.dropTable(self.indexDb, metaTable, mustExist=False)
+        #self.czarWmgr.dropTable(self.indexDb, metaTable, mustExist=False)
```

Comment:
I'm somewhat worried about duplicate records if table is not dropped. If you do incremental data uploads and update secondary index every time then it will try to insert duplicate records into secondary index. With our current implementation duplicate records are ignored, but this is most likely just a happy coincidence and not a part of a design (design was to drop table to avoid trouble). We should keep this in mind when we update our data loading procedures.